pkg/sfu/videolayerselector/vp9.go

```diff
@@ -93,7 +102,7 @@ func (v *VP9) Select(extPkt *buffer.ExtPacket, _layer int32) (result VideoLayerS
 	}
 
 	result.RTPMarker = extPkt.Packet.Marker
-	if extPkt.VideoLayer.Spatial == v.currentLayer.Spatial && vp9.E {
+	if vp9.E && extPkt.VideoLayer.Spatial == currentLayer.Spatial && (vp9.P || v.targetLayer.Spatial <= v.currentLayer.Spatial) {
```

Comment:
This is the stutter fix. When `vp9.P` is `true`, cannot scale up at this picture. So, set the marker as long  incoming spatial layer is the current spatial layer and scaling cannot happen at this picture.